.githooks/pre-commit/no-single-quote

```diff
@@ -1,10 +0,0 @@
-#!/bin/sh
```

Comment:
Can we keep this hook? I kind of like not having to think about quotes, and just disallow single quote in those files.
After this PR in, perhaps I'll check if the `install.sh` can also use double-quotes only.

Reply:
Sure. !